src/store/WalletStore.ts

```diff
@@ -1,15 +1,20 @@
 import { defineStore } from "pinia";
 import { Wallet, WalletStoreState } from "../types";
+import { WebMode } from "@algo-builder/web";
 
 export default defineStore("WalletStore", {
 	state: (): WalletStoreState => {
 		return {
 			walletKind: Wallet.NONE,
+			webMode: <WebMode>{},
 		};
 	},
 	actions: {
 		setHasAlgoSigner(value: Wallet) {
 			this.walletKind = value;
 		},
+		setWebMode(value: WebMode) {
+			console.log("WebMode Initialized", value);
+		},
```

Comment:
```suggestion
		setWebMode(m: WebMode) {
			console.log("WebMode Initialized", m);
		},
```

Reply:
`value` variable name is too generic in this context, we can find better names.

Reply:
Ok. I will make it webMode

Reply:
I think we don't need to repeat the type, `m` or `mode` is enough, but as  you prefer.